### PR TITLE
Add example for call command

### PR DESCRIPTION
**Pull request by maksimdrachov**
Question:

Is there a way to this:

```
yakut call 125 435:uavcan.node.ExecuteCommand '{command: COMMAND_RESTART, parameter: "firmware.bin"}'
```

Instead of this:

```
yakut call 125 435:uavcan.node.ExecuteCommand '{command: 65533, parameter: "firmware.bin"}'
```

[Relevant forum question](https://forum.opencyphal.org/t/yakut-file-commands/1815/2)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -305,6 +305,12 @@ You can still override the type if you want to use a different one (e.g., if the
 y q 42 least_squares:my_namespace.MySpecialType '[[10, 1], [20, 2]]'
 ```
 
+If your service call requires multiple arguments, specify them as follows:
+
+```bash
+yakut call 125 435:uavcan.node.ExecuteCommand '{command: 65533, parameter: "firmware.bin"}'
+```
```
Comment (pavel-kirienko) [inline README.md:312]: Could you please check that this is also valid (by design it should be) while you're at it? Maybe it also should be added to the help output.

```suggestion
If your service call requires multiple arguments, specify them as follows:

```bash
yakut call 125 435:uavcan.node.ExecuteCommand '{command: 65533, parameter: "firmware.bin"}'
yakut call 125 435:uavcan.node.ExecuteCommand '[65533, "firmware.bin"]'  # Shorter but position-sensitive
```
```

**Review by pavel-kirienko — APPROVED** [review]
Please check my comment

**Comment by pavel-kirienko** [comment]
Oh also please bump the patch version

File: README.md (hunk 1 above)
Comment (maksimdrachov) [reply README.md:312]: Yes it works.

**Pull request merged**